content/pages/vocational-rehab-and-employment/apply-vre.md

```diff
@@ -48,59 +33,63 @@ You may also qualify for VR&amp;E benefits if you're a Veteran who was discharge
 
 </div>
 
-<div markdown="1">
-
 ### What kind of VR&amp;E benefits can I get?
 
 - Job search and placement assistance
 - Vocational (job) training 
 - Education resources and counseling
 - Adaptive-housing counseling and referrals
 
-</div>
-
-<div markdown="1">
-
 <hr>
 
 ### How do I get these benefits?
-</div>
+
 You'll need to apply for benefits. Follow these steps to apply:
 
-<ol class="process" markdown="0">
-<li class="step one wow fadeIn animated" markdown="1">
+<ol class="process">
+
+<li class="step one">
+
 [Log in to your eBenefits account](https://www.ebenefits.va.gov/ebenefits/homepage).
+
 </li>
 
-<li class="step two wow fadeIn animated" markdown="1">
+<li class="step two">
```

Comment:
For the class names for subway map steps, this is all you should need: `<li class="step {number}">` and if it's the last step, `<li class="step last {number}">`